src/modules/projects/api/update-project-mutation.ts

```diff
@@ -19,7 +23,17 @@ export const UPDATE_PROJECT = gql`
 `;
 
 export const useProjectUpdate = () => {
+    const { [RouteParams.ProjectId]: projectId } = useParams();
+
+    const { showNotification } = useNotificationContext();
+
     return useMutation<UpdateProjectResult, { project: UpdateProjectInput }>(UPDATE_PROJECT, {
-        refetchQueries: [GET_PROJECTS_QUERY],
+        refetchQueries: [
+            GET_PROJECTS_QUERY,
+            { query: GET_PROJECT_QUERY, variables: { projectId } },
+        ],
+        onError: (error) => {
+            showNotification("error", error.message);
+        },
```

Comment:
maybe add onCompleted here to show notification that the project have been successfully updated?

Reply:
well, haven't seen any "on success" notifications in example app (only "on error"), so why add those here?